Dockerfile

```
@@ -0,0 +1,14 @@
+FROM golang:1.20 AS builder
```

Consider using the alpine image in your builder if it's being used as the runner. There can be slight discrepancies in OS support so you should maintain the same base for each 

```suggestion
FROM golang:1.20-alpine AS builder
```